app/controllers/status_controller.rb

```diff
@@ -1,10 +1,16 @@
 class StatusController < ApplicationController
+  # This is used by the liveness check, used to see if a pod needs replacing
```

Comment:
I wonder if we could or should rename any of this stuff (stuff being method names and paths) to make it better fit our mental models

Reply:
Oh the other PR gets rid of one half anyway so that's fine